apps/asap-server/src/controllers/teams.ts

```diff
@@ -122,7 +122,21 @@ const fetchUsers = async (id: string, client: Got): Promise<RestUser[]> => {
   return res ? res.items : [];
 };
 
-export default class Teams {
+export interface TeamController {
+  update: (id: string, tools: TeamTool[], user: User) => Promise<TeamResponse>;
+  fetch: (
+    options: {
```

Comment:
Now would be a good time to expose those `FetchOptions` to the other controllers and add reuse them for this PR. What do you think?